### ISSUE TITLE
Window is bigger every time

### ISSUE THREAD

**Issue by FrouxBY** [jellyfin/jellyfin-mpv-shim#207]
**Describe the bug**
Every time I re-open Jellyfin Desktop, the window size is increased compared to the previous time, even resulting in the window becoming larger than the monitor sometimes.

**To Reproduce**
1. fresh install on windows
2. Having a computer with 2 monitor: 1080p (laptop) + 4K (external on GPU)
3. Put laptop monitor to 100% scale and 4k external monitor to 150% scale 
4. open and reduce the jellyfin window to a small size, then close and open again jellyfin desktop to see the windows is now bigger (repeat)

**Expected behavior**
The window should stay the same size as the last time I closed it.

**Desktop (please complete the following information):**
 - OS:  Windows
 - Version 1.10.4

**My investigation**
- This does seem to happen only when the 4k screen is set to 150% scale. If I put it to 100% scale, then the jellyfin does not grow each reopening. 
- It happens on both screen, even on the 1080p 100% scale (when the other one is set to 150%)

Thanks for your help ! If you need any test/logs/whatever, I be pleased to help solve this issue

**Comment by iwalton3** [comment]
Similar to https://github.com/jellyfin/jellyfin-desktop/issues/162.

Please post detailed instructions on what is happening including the contents of the geometry config file between each open/close.

**Comment by FrouxBY** [comment]
Yes, I actually miss the other issue when searching for it, sorry for the duplicate.

So here is more details on what is happening:
- conf.json relevant options, I tried different options ( see | ) :
```"desktop_fullscreen": false,
    "desktop_keep_pos": false | true,
    "desktop_keep_size": true,
    "desktop_scale": 1 | 1.25 | 1.5,
```

Now I am on scale 150% (in OS option) on my 3840*2160  monitor and whatever desktop_scale in conf.json (does not have an influence)

The initial value of layout.json is :
`{"x": 1021, "y": 380, "width": 1759, "height": 1096}`

Then I open Jellyfin Desktop, and the window seems to be 1759*1096 pixel and centered (1021 + 1759 + 1021 ~ 3840 )
Layout.json has not yet changed, I do not move or resize the window, and closed it.

then, layout.json is the following: 
`{"x": 606, "y": 242, "width": 2628, "height": 1616}`

Wich is about 1.5 of the previous width/height 

If I opened it again, the window match this new layout and we still have a centered window (606+2628+606 ~ 3840)

if I do it again, then I hit the height limit i guess, and I get {"x": 0, "y": 0, "width": 3866, "height": 2186}

Experiments with other scale (on OS display manager) show that jellyfin display correctly with the value stored on layout.json on the oppening but save **scale*displayed layout** on jellyfin closure, which lead to the issue.

I hope this help
